specs/daito/alibaba-provider.md

```diff
@@ -0,0 +1,327 @@
+# Integration of Alibaba Cloud as a backend in OpenSDS/Multi-cloud
+**Author(s)**:  [Click2Cloud Technology Services India Pvt. Ltd.](https://github.com/Click2Cloud)
+### Summary
+Using this feature, user will be able to do object management mechanism and migration for Alibaba Cloud in multi-cloud .
+### Motivation
+In existing OpenSDS Gelato project, we don't have any option using which user can manage object management mechanism and migration for Alibaba Cloud. So we adding  alibaba as backend for Object management and migration.
+### Goals
+
+* Add support of Alibaba OSS service in OpenSDS Gelato Project.
+
+
+
+### Design Details
```

Comment:
This link is broken. Please check

Reply:
@Click2Cloud-Gamma Can you please this part?

Reply:
Clicking on the picture gives 404 error

Reply:
This issue occured due to github functionality, after merging PR image can be viewed in file too.
Thanks